docs/quickstart.rst

```diff
@@ -19,6 +19,11 @@ Start by creating an account in FEDn Studio and set up a project by following th
 Install FEDn
 ------------
 
+Client node: A physical or virtual machine (MacOSx Linux or Windows) used to perform initial steps.
```

Comment:
this just makes it more complex for the user I think, Also, it sound to much "hadoop"